app/Http/Controllers/AlbumController.php

```diff
@@ -251,8 +251,21 @@ public function getPublic(Request $request)
 					if ($this->sessionFunctions->has_visible_album($album->id)) {
 						return 'true';
 					}
-					if ($album->password == '' || Hash::check($request['password'], $album->password)) {
```

Comment:
I take it the `$album->password == '' || ` part is handled elsewhere so not required here?

Reply:
Yeah, exactly six lines earlier...